internal/inputs/http.go

```diff
@@ -18,10 +18,10 @@ import (
 	"strings"
 	"time"
 
+	xj "github.com/basgys/goxml2json"
```

Comment:
@haihongren we need to add this package to this file -> https://github.com/newrelic/nri-flex/blob/master/THIRD_PARTY_NOTICES.md

Reply:
done!tx